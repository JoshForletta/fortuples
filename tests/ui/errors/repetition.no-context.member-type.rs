use fortuples::fortuples;

struct Container<T>(T);

fortuples! {
    impl Container<#Tuple> {
        #Member
    }
}

fn main() {}
