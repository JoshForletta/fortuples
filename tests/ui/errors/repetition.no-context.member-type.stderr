error: Attempting to expand the tuple member types without the repetition context.
       Try rewrite this like the following: `#(#Member),*`
 --> tests/ui/errors/repetition.no-context.member-type.rs:7:10
  |
7 |         #Member
  |          ^^^^^^
